spring-stack/requirements.yaml

```diff
@@ -1,25 +1,30 @@
 dependencies:
   - name: spring-config-server
-    repository: https://raw.githubusercontent.com/ibm-cloud-architecture/refarch-cloudnative-spring/master/docs/charts
+    repository: http://raw.githubusercontent.com/ibm-cloud-architecture/refarch-cloudnative-spring/master/docs/charts
```

Comment:
@jkwong888 super nitty-gritty but can we make sure we have this as **https** vs **http** to stay consistent with index.yaml?